MAUI/DataForm/Editors.md

```diff
@@ -291,6 +319,107 @@ public string Password { get; set; }
 
 ![Password editor in .NET MAUI DataForm.](images/editors/dataform-password-editor.png)
 
+## Numeric editor
+In the numeric editor, the `SfNumericEntry` will be loaded and DataForm Numeric editor supports `int`, `double`, `float` data type property.
+
+Also, to add a DataForm Numeric editor, register the editor as `DataFormEditorType.Numeric` for the required property using the [RegisterEditor](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.SfDataForm.html#Syncfusion_Maui_DataForm_SfDataForm_RegisterEditor_System_String_Syncfusion_Maui_DataForm_DataFormEditorType_) method.
+
+#### Changing the Numeric editor properties
+You can use `GenerateDataFormItem` event to change Numeric editor properties
+
+
+{% tabs %}
+{% highlight C# %}
+
+public string Amount { get; set; }
```

Comment:
remove string  
public string Amount { get; set; }

for custom editor you can explain int type

Reply:
changed as int